lambda_functions/crud/operations.py

```diff
@@ -295,3 +331,43 @@ def get_all_review_questions_db():
     review_questions = session.query(ReviewQuestion).all()
     return review_questions
 
+
+def get_open_item_for_user_db(user):
+    """Inserts a new review answer into the database
```

Comment:
This description does not fit

Reply:
I cannot open a PR for comment changes for some reason. I will include this change in my next "big" PR.